projects/L4T/packages/tegra-bsp/package.mk

```diff
@@ -123,21 +154,32 @@ fi
 makeinstall_target() {
   build_install
   cp -PRv install/* $INSTALL/
-  cp -PRv install/usr/lib/* ${TOOLCHAIN}/aarch64-libreelec-linux-gnueabi/sysroot/usr/lib/
+  cp -PRvn install/usr/lib/* ${TOOLCHAIN}/aarch64-libreelec-linux-gnueabi/sysroot/usr/lib/
+  #install multimedia_api_headers
+  cp -PRvn multimedia_api/usr/src/jetson_multimedia_api/include/* ${SYSROOT_PREFIX}/usr/include
+  mkdir -p ${SYSROOT_PREFIX}/usr/src/jetson_multimedia_api/samples/common
+  cp -PRn multimedia_api/usr/src/jetson_multimedia_api/samples/common/*  ${SYSROOT_PREFIX}/usr/src/jetson_multimedia_api/samples/common/
   PWD=$(pwd)
   cd ${TOOLCHAIN}/aarch64-libreelec-linux-gnueabi/sysroot/usr/lib/
   rm libv4lconvert.so.0 libv4lconvert.so libv4l2.so.0 libv4l2.so
   mv libv4l2.so.0.0.999999 libv4l2.so.0
   cp libv4l2.so.0 libv4l2.so
   mv libv4lconvert.so.0.0.999999 libv4lconvert.so.0
   cp libv4lconvert.so.0 libv4lconvert.so
+  rm libgbm.so.1
+  cp libnvgbm.so libgbm.so.1
+  #cp libnvgbm.so libgbm.so
+  cp libnvidia-egl-wayland.so libvnidia-egl-wayland.so.1
+  rm libdrm.so.2
+  cp libdrm_nvdc.so libdrm.so.2
   cd $PWD
-  if [ "$DEVICE" = "Switch" ]; then
+
+  if [ "$DEVICE"=="Switch" ]; then
```

Comment:
single equal + spaces around equal

Reply:
Yeah, sorry, I copied that from a different old branch, from pre merging the switch stuff upstream, this file is actually an old version from one of my old vulkan testing branches with a lot of minor modifications to fix decoder stuff, and switch to proper glvnd, as that is how nvidia does it in ubuntu. Also, egl libs being set up properly is required for the decoder, so I didnt have much choice but to change the bsp package, same can be said with the other